hub/api/tensor.py

```diff
@@ -212,6 +216,18 @@ def numpy(self, aslist=False) -> Union[np.ndarray, List[np.ndarray]]:
 
         return self.chunk_engine.numpy(self.index, aslist=aslist)
 
+    def plot(self):
+        """Visualize a single sample of this tensor. Currently only supports single-image visualization.
+
+        Example:
+            >>> images = ds.images
+            >>> images[0].shape
+            (100, 100, 3)
+            >>> images[0].plot()  # will bring up a matplotlib imshow plot!
+        """
+
+        visualize_tensor(self)
```

Comment:
I don't like passing `self` around to helper methods. I prefer to pass the necessary member variables instead. 

It's hard to manage circular dependencies this way.